advntr/vntr_finder.py

```diff
@@ -499,7 +502,16 @@ def get_dominant_copy_numbers_from_spanning_reads(self, spanning_reads):
             if logp < rev_logp:
                 vpath = rev_vpath
             observed_copy_numbers.append(get_number_of_repeats_in_vpath(vpath))
-
+            if log_pacbio_reads:
+                repeats = get_number_of_repeats_in_vpath(vpath)
```

Comment:
We can call `get_number_of_repeats_in_vatph(vpath)` once.

Reply:
Great point. Fixed it.